### PR TITLE
Change moderation reasons

### LINKED ISSUE #758: "Not an FOI Request" wording is no longer generic
Before we upgraded we didn't specify the reason why a request was hidden. Now we do:

Example Request: https://www.righttoknow.org.au/request/train_footage_from_the_151220
image

Ideally we should say something like 

> "This request has been hidden by an Administrator as it's a request for personal information".

### PR DESCRIPTION
**Pull request by benrfairless**
This PR resolves #758 and makes everything generic again.

It also updates the wording in the admin back-end so it's more like what I would normally manually type to someone:

image

### CODE REVIEW

File: lib/views/help/house_rules.html.erb [hunk 1]
```diff
@@ -67,9 +67,9 @@
   </ul>
   <h2>What happens if you break these rules</h2>
   <p>
-    If you break any of the rules we will likely suspend your account. This means you will no longer be able to make new requests or
-    updates on <%= site_name %>. We also may
-    remove your existing requests and annotations.
+    If you break any of the rules there are a number of actions we may take.
```
Comment (mlandauer) [inline lib/views/help/house_rules.html.erb:70]: It's confusing having this commit included in this PR when it's not part of the change that you're talking about

**Review by mlandauer — COMMENTED** [review]
It's also a bit confusing that the change in the first commit is remove a "msgid" and adding a "msgid" and "msgstr". I would always expect msgid and msgstr to always go together and be added or removed as a pair

**Comment by mlandauer** [comment]
I think given the extra commit and also that it's conflicting with staging any chance @benrfairless you could redo this PR and address the comments?

Thanks!

**Comment by benrfairless** [comment]
I'm going to close this PR and start again I think.
